src/nomad_simulation_parsers/parsers/lammps/parser.py

```diff
@@ -0,0 +1,1485 @@
+            return {keys[i]: values[i] for i in range(len(keys))}
+
+        self._quantities = [
+            Quantity(
+                'time_step',
+                r'\s*ITEM:\s*TIMESTEP\s*\n\s*(\d+)\s*\n',
+                comment='#',
+                repeats=True,
+            ),
+            Quantity(
+                'n_atoms',
+                r'\s*ITEM:\s*NUMBER OF ATOMS\s*\n\s*(\d+)\s*\n',
+                comment='#',
+                repeats=True,
+            ),
+            Quantity(
+                'pbc_cell',
+                # TODO: LB - Check why pbc none for atom_run
+                r'\s*ITEM: BOX BOUNDS\s*([\s\w]+)\n([\+\-\d\.eE\s]+)\n',
+                str_operation=self.get_pbc_cell,
+                comment='#',
+                repeats=True,
+            ),
+            Quantity(
+                'atoms_info',
+                r'\s*ITEM:\s*ATOMS\s*([ \w]+\n)*?([\+\-eE\d\.\n ]+)',
+                str_operation=get_atoms_info,
+                comment='#',
+                repeats=True,
+            ),
+        ]
+
+    @property
+    def with_trajectory(self) -> bool:
+        return self.get('atoms_info') is not None
+
+    @property
+    def n_frames(self) -> int:
+        return len(self.get('atoms_info', []))
+
+    @property
+    def masses(self) -> np.ndarray:
+        return self._masses
+
+    # TODO: handle non-atomistic representations
+    @masses.setter
+    def masses(self, val):
+        self._masses = val
+        if self._masses is None:
+            return
+
+        self._masses = val
+        if self._chemical_symbols is None:
+            masses = self._masses[0][1]
+            self._chemical_symbols = {}
+            for i in range(len(masses)):
+                symbol_idx = np.argmin(
+                    abs(self._reference_masses['masses'] - masses[i][1])
+                )
+                self._chemical_symbols[masses[i][0]] = self._reference_masses[
+                    'symbols'
+                ][symbol_idx]
+
+    def get_atom_labels(self, idx):
+        atoms_info = self.get('atoms_info')
+        if atoms_info is None:
+            return
+
+        atoms_id = atoms_info[idx].get('id')
+        default = ['CGX' for _ in atoms_id] if atoms_id is not None else None
+        atoms_type = atoms_info[idx].get('type')
+        if atoms_type is None:
+            return default
+        if self._chemical_symbols is None:
+            return default
+
+        try:
+            atom_labels = [self._chemical_symbols[atype] for atype in atoms_type]
+        except Exception:
+            self.logger.error('Error resolving atom labels.')
+            return
+
+        return atom_labels
+
+    def get_positions(self, idx):
+        atoms_info = self.get('atoms_info')
+        if atoms_info is None:
+            return
+
+        atoms_info = atoms_info[idx]
+
+        cell = self.get('pbc_cell')
+        cell = None if cell is None else cell[idx][1]
```

Comment:
shouldnt you have `if is cell is None:
return` here?

Reply:
ah you have the l349 case, put this as the first case then all that requires cell in elif and return if cell is None